### ISSUE TITLE
custom.list only replicates once it exists on secondary

### ISSUE THREAD

**Issue by xela1** [vmstan/gravity-sync#39]
**Describe the Issue**
The custom.list only gets pulled from the primary, if it already exists on the secondary server.

**Screenshots**
[ INFO ] _remote_ has custom.list
[ INFO ] No custom.list Detected on _local_
[ INFO ] No Replication Required
[ INFO ] Gravity Sync PULL Aborting (4 seconds)

sudo touch custom.list

[  OK  ] Analyzing gravity.db on _remote_
[  OK  ] Analyzing gravity.db on _local_
[  OK  ] Analyzing custom.list on _remote_
[  OK  ] Analyzing custom.list on _local_
[ WARN ] Differenced custom.list Detected
[ WARN ] Replication Required

**Configuration:**
 - OS for Primary/Secondary: Raspbian Buster lite
 - Platform for Primary/Secondary: Raspberry Pi3/Raspberry Pi Zero W
 - SSH Server/Client: OpenSSH
 - SSH Authentication: Keypair
 - Gravity Sync Version: 1.8.1
 - Pi-hole Versions: v5.0

**Additional Context**
Add any other context about the problem here. If you have any advanced flags set in your `gravity-sync.conf` please list them here.

**Comment by vmstan** [comment]
I think I've been able to reproduce this. It seems like it pulls and creates a new one if the gravity.db has a change, but if it only detects the custom.list doesn't exist, it doesn't proceed to the pull. I wasn't thinking through a situation where that would be the only thing that would need to be replicated, but if you've just made the change to the primary custom.list and nothing else, it wouldn't get there until the other piece of the puzzle was changed. Which you wouldn't know to do.

I've added a `HASHMARK=$((HASHMARK+1)` to the function that does this detection, basically treating it as a difference in comparison (typically it would do an MD5 hash between the two files, but if the file is just missing, it's still a difference) -- I'll test this and a couple other tweaks and likely roll 1.8.2 out soon.
